pkg/solana/config_tracker.go

```diff
@@ -5,13 +5,11 @@ import (
 
 	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
 	"github.com/smartcontractkit/libocr/offchainreporting2/types"
-
-	"github.com/smartcontractkit/chainlink-solana/pkg/solana/client"
 )
 
 type ConfigTracker struct {
 	stateCache *StateCache
-	reader     client.Reader
+	reader     GetReader
```

Comment:
```suggestion
	getReader     GetReader
```

other places also use `getReader` - naming is also clearer